apps/src/aichat/constants.ts

```diff
@@ -1 +1,14 @@
 export const CHAT_COMPLETION_URL = '/openai/chat_completion';
+import {ChatCompletionMessage, Role, Status} from './types';
+
+const initialChatMessages: ChatCompletionMessage[] = [
+  {
+    id: 1,
+    role: Role.ASSISTANT,
+    chatMessageText:
+      'Welcome to AI Chat! I am your assistant - please ask me questions according to the instructions given.',
```

Comment:
Probably a question for product/curriculum - do we know if this initial message is meant to be hardcoded, or set by levelbuilders? Or generated by open AI?

Reply:
Thanks! Just checked with prod/curric in [this Slack thread](https://codedotorg.slack.com/archives/C052GPUCRNZ/p1692805333308489) and will implement their suggestions/preferences.